### ISSUE TITLE
Sleeping Skips multiple days

### ISSUE THREAD

**Issue by Sataikoyurai** [anegostudios/VintageStory-Issues#841]
**Game Version:** 1.14.8
**Platform:** Windows
**Modded:** No

### Description
When sleeping in the normal bed and watching the time I noticed multiple days passing instead of 8.5 hours like its supposed to 

### How to reproduce
1. Sleep in bed
2. Open character menu
3. Shift out of sleep (not sure if this part is necessary)

### Expected behavior
Sleep for 8.5 hours

**Comment by tyronx** [comment]
I can not reproduce this on default settings. Did you set any custom day length or time speed value?

**Comment by Ginganord** [comment]
**Game Version:** 1.15.5 (Stable)
**Platform:** Windows 10
**Modded:** No
Playing LAN Multiplayer (Host) through LogMeIn Hamachi VPN

**Description**
Hello and sorry - I know you already closed this, but I just had the same issue happen to me.

Using 7-hour beds, a week had passed.  The game told me and friend that host connection failed (so probably connection related, though I maintained consistent voice contact with friend during duration).  We could not stop sleeping until I closed the game.  Upon reopening, only a few days had actually passed, with satiety still lost, but food in stores maintaining original expected spoilage time.

Later happened in a significantly different way (same preexisting conditions).  Only a few extra in-game hours passed, for a total of 10 hours slept.  Host connection issue message never appeared.  We seemingly only woke up upon us both attempting to wake, but that is inconclusive because there was a time delay (we attempted to wake upon 7-hours' passage).

Next sleep cycle functioned as intended, with the game automatically waking us at 7 hours.

All world settings are set to default.
